packages/admin/admin-date-time/src/utils/datePickerHelpers.ts

```diff
@@ -3,3 +3,10 @@ defaultMinDate.setFullYear(defaultMinDate.getFullYear() - 120);
 
 export const defaultMaxDate = new Date();
 defaultMaxDate.setFullYear(defaultMaxDate.getFullYear() + 40);
+
+export const getDateString = (date: Date) => {
+    const year = date.getFullYear();
+    const month = date.getMonth() + 1;
+    const day = date.getDate();
+    return `${year}-${month < 10 ? "0" : ""}${month}-${day < 10 ? "0" : ""}${day}`;
+};
```

Comment:
date-fns is already a dependency, so you can use `format` for this (which also handles the 0-based month)

https://date-fns.org/v3.6.0/docs/format